src/sagemaker/jumpstart/hub/utils.py

```diff
@@ -193,7 +193,11 @@ def get_hub_model_version(
     hub_model_version: Optional[str] = None,
     sagemaker_session: Session = constants.DEFAULT_JUMPSTART_SAGEMAKER_SESSION,
 ) -> str:
-    """Returns available Jumpstart hub model version"""
+    """Returns available Jumpstart hub model version
+
+    Raises:
+        ResourceNotFound: If the specified model is not found in the hub.
```

Comment:
the exception type should be a Python class, not the error returned by the API.
Doesn't this method raise or `Exception`, `KeyError` or potentially a `ClientException` on line 203?

Reply:
yes my bad, updated it to ClientError exception instead